internal/config/mount_config.go

```diff
@@ -219,5 +226,13 @@ func NewMountConfig() *MountConfig {
 		MaxRetryAttempts: DefaultMaxRetryAttempts,
 	}
 
+	mountConfig.GCSRetries = GCSRetries{
```

Comment:
@songjiaxun I think these changes are not part of this PR.

Reply:
Good catch! Done.